cogbot/extensions/mcnbtdoc.py

```diff
@@ -0,0 +1,915 @@
+                await self.bot.add_reaction(ctx.message, u'🤷‍♀️')
+                return
+            else:
+                await self.bot.send_message(
+                    ctx.message.channel,
+                    'Found {} matches:\n{}{}'.format(
+                        len(matches),
+                        '\n'.join([' * {} in `{}` with type {}'.format(
+                            fn,
+                            '::'.join(cn),
+                            format_nbttype(fv['nbttype'], data).replace('\n', '\n    ')
+                        ) for cn, fn, fv in matches[:self.config.field_limit]]),
+                        '\nAnd {} more'.format(len(matches) - self.config.field_limit)
+                            if len(matches) > self.config.field_limit else ''
+                    )
+                )
+                return
+        else:
+            if not args['type'] or not args['name']:
+                await self.bot.add_reaction(ctx.message, u'💩')
+                return
+
+            if args['path']:
+                path = args['path'].split('.')
+            else:
+                path = []
+
+            if not args['name'].startswith('minecraft:'):
+                name = 'minecraft:{}'.format(args['name'])
+            else:
+                name = args['name']
+
+            item = await self.get_from_reg(
+                name,
+                'minecraft:{}'.format(args['type']),
+                ctx,
+                path,
+                version=args['version']
+            )
+            title = name
+            if len(path) > 0:
+                title = path[-1]
+        ace = ActiveEmbed(None, item, data, self.config.field_limit, title, False, self.config.active_limit)
+        if ace.should_scroll():
+            msg = 'page {}'.format(ace.get_page_msg())
+        else:
+            msg = ''
+        em = ace.get_embed()
+        if em == None:
+            await self.bot.add_reaction(ctx.message, u'❌')
+            return
+        thismsg = await self.bot.send_message(ctx.message.channel, content=msg, embed=em)
+        ace.message = thismsg
+        if not thismsg.channel.server in self.active_embeds:
+            self.active_embeds[thismsg.channel.server] = {}
+        self.active_embeds[thismsg.channel.server][str(thismsg.id)] = ace
+        ace.remove_task_factory = lambda: remove_ae(
+            str(thismsg.id),
+            thismsg.channel.server,
+            self,
+            self.config.active_limit
+        )
+        ace.change_active()
+        if ace.should_scroll():
+            ace.is_scrolling = True
+            await self.bot.add_reaction(thismsg, u'◀')
+            await self.bot.add_reaction(thismsg, u'▶')
+        await self.bot.add_reaction(thismsg, u'🔼')
+        await self.bot.add_reaction(thismsg, u'🔽')
+
+    @checks.is_manager()
+    @commands.command(pass_context=True, name='nbtreload', hidden=True)
+    async def cmd_invitereload(self, ctx: Context):
```

Comment:
This should probably be `cmd_nbtreload` just for readability sake. (I probably forgot to update it myself in the original nbtpath extension.)